core/unit_test/CMakeLists.txt

```diff
@@ -367,6 +367,16 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;OpenACC;HIP;SYCL)
         configure_file(${dir}/dummy.cpp ${file})
         list(APPEND ${Tag}_VIEWSUPPORT ${file})
       endforeach()
+      if(NOT Kokkos_ENABLE_IMPL_VIEW_LEGACY)
+        foreach(Name ViewCustomizationAccessorArg)
+          set(file ${dir}/Test${Tag}_View_${Name}.cpp)
+          # Write to a temporary intermediate file and call configure_file to avoid
+          # updating timestamps triggering unnecessary rebuilds on subsequent cmake runs.
+          file(WRITE ${dir}/dummy.cpp "#include <Test${Tag}_Category.hpp>\n" "#include <view/Test${Name}.hpp>\n")
+          configure_file(${dir}/dummy.cpp ${file})
+          list(APPEND ${Tag}_VIEWSUPPORT ${file})
```

Comment:
Instead of duplicating that block, why are we not using a variable for the list of `Name`s to iterate over?